test/python/conftest.py

```diff
@@ -16,99 +16,14 @@ def module_tmpdir(request):
 
 @pytest.fixture(scope="module")
 def project_path(module_tmpdir):
-    return create_project(module_tmpdir, project_name, 'cartridge')
+    path = create_project(module_tmpdir, project_name, 'cartridge')
 
+    # add cartridge.post-build file to remove test/ and tmp/ contents
+    with open(os.path.join(path, 'cartridge.post-build'), 'w') as f:
```

Comment:
Where is a test for "pre-build"?

Reply:
This diff is outdated, I just checked that this test is here.